modules/api/src/Validators/ITransparencySiteRequestValidator.cs

```diff
@@ -4,5 +4,7 @@ namespace GpConnect.NationalDataSharingPortal.Api.Validators;
 
 public interface ITransparencySiteRequestValidator
```

Comment:
Can we stick all the interfaces in a separate folder to keep them separate from the implementations?

Reply:
Interaces migrated as part of fa51e77b6f905ef50ab35d3a0ce9d1f10cc40f8e, all tests continue to pass